pennylane/ops/op_math/controlled_ops.py

```diff
@@ -1421,17 +1445,10 @@ def __init__(
         wires: WiresLike = (),
         control_values: Union[bool, List[bool], int, List[int]] = None,
         work_wires: WiresLike = (),
-        work_wire_type: Literal["clean", "dirty"] = "clean",
+        work_wire_type: Literal["clean", "dirty"] = "dirty",
     ):
-        wires = Wires(() if wires is None else wires)
-        work_wires = Wires(() if work_wires is None else work_wires)
-
-        if work_wire_type not in {"clean", "dirty"}:
-            raise ValueError(
-                f"work_wire_type must be either 'clean' or 'dirty'. Got '{work_wire_type}'."
-            )
-        self.work_wire_type = work_wire_type
-
+        wires = wires or ()
+        work_wires = work_wires or ()
```

Comment:
This will cause trouble with `wires=0`, since that is falsey.

Reply:
Updated all of them (I think I got them all)

Reply:
I found one more since it was using an empty list instead of tuple, but I didn't find any other instances either 😄